_inc/idc-notice.js

```diff
@@ -46,4 +53,28 @@
 	function fixJetpackConnection() {
 		notice.addClass( 'jp-idc-show-second-step' );
 	}
+
+	/**
+	 * This function will fire both a Tracks and MC stat.
+	 * It will make sure to format the event name properly for the given stat home.
+	 *
+	 * Tracks: Will be prefixed by 'jetpack_idc_' and use underscores.
+	 * MC: Will not be prefixed, and will use dashes.
+	 *
+	 * @param eventName string
+	 */
+	function trackAndBumpMCStats( eventName ) {
+		if ( 'undefined' !== eventName && eventName.length ) {
+
+			// Format for Tracks
+			eventName = eventName.replace( /-/g, '_' );
+			eventName = eventName.indexOf( 'jetpack_idc_' ) !== 0 ? 'jetpack_idc_' + eventName : eventName;
+			analytics.tracks.recordEvent( eventName, {} );
+
+			// Now format for MC stats
```

Comment:
No, but either are accurate :) 

`Now we format it for mc stats...`
`this is a new format for mc stats!`